.github/Bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+**Symfony version(s) affected**: x.y.z
```

Comment:
i am not sure we should have this. we could talk about the library version, but i would just leave this out